docs/querying/query-context.md

```diff
@@ -79,6 +79,15 @@ In addition, some query types offer context parameters specific to that query ty
 |-----------------|---------------------|----------------------|
 |skipEmptyBuckets | `false`             | Disable timeseries zero-filling behavior, so only buckets with results will be returned. |
 
+### Join filter
+
+|property         |default              | description          |
+|-----------------|---------------------|----------------------|
+|enableJoinFilterPushDown | `true` | Controls whether a join query will attempt filter push down, which reduces the number of rows that have to be compared in a join operation.|
+|enableJoinFilterRewrite | `false` | Controls whether filter clauses that reference non-base table columns will be rewritten into filters on base table columns.|
+|enableJoinFilterRewriteValueColumnFilters | `false` | Controls whether Druid rewrites non-base table filters on non-key columns. Requires a scan of the non-base table.|
```

Comment:
Suggest "Controls whether Druid rewrites non-base table filters on non-key columns in the non-base table. Requires a scan of the non-base table."